architecture-decision-records/0011-base25-encoding-for-consignment-reference.md

```diff
@@ -0,0 +1,125 @@
+# base25 encoding for consignment references
+
+**Date:** 2020-11-13
+
+## Context
+
+As part of the 'transfer summary' page, the user will see a 'consignment reference' to identify their consignment (which can be useful if a user needs to contact us about a specific consignment).
+The user will also be able to see the progress of multiple consignments on the 'transfer history' page, which will use the consignment reference to identify separate transfers.
+The TDR system already generates a UUID when a consignment is created, but this can be difficult for users to read or work with.
+
+Utilising the `SeriesID` was considered to show the users, but one of the long-term goals for TDR is to remove the concepts of series', so introducing the user to these before removing them did not seem sensible.
+
+Discussions were also had about using the transferring body/code to identify a consignment, but was decided against due to potential for these to change.
+
+The TDR team decided that it would be helpful to generate a separate reference for users to use to refer to consignments using a base25 alphabet.
+With the introduction of the base25, the TDR backend will also be able to use either the UUID or consignment reference to look up consignments, but users will exclusively see the consignment reference.
```

Comment:
I think it's worth clarifying that the users will see consignment references on TDR pages, but the URLs will continue to use the UUIDs. (Just in case someone in future notices the URLs and is confused because they are technically user-visible.)

Reply:
Ooh good point! I'll change that :+1: